File: program44.cpp
#include<iostream>
using namespace std;

bool CheckPrime(int iNo)
{
    int iCnt = 0;
    bool bFlag = true;

    for(iCnt = 2; iCnt < iNo; iCnt++)
    {
        if((iNo % iCnt) == 0)
        {
            bFlag = false;
            break;
        }
    }
    return bFlag;

}

int main()
{
    int iValue = 0;
    bool bRet = false;

    cout<<"Enter the number :"<<endl;
    cin>>iValue;

    bRet = CheckPrime(iValue);

    if(bRet == true)
    {
        cout<<"number is prime "<<endl;
    }
    else
    {
        cout<<"number is not prime "<<endl;
    }   

    return 0;
}

File: program44.java
import java.util.*;

class Arithematic
{
    public int iNo1;

    public Arithematic(int A)
    {
        iNo = A;
    }
    public boolean CheckPrime()
    {
        int iCnt = 0;
        boolean bFlag = true;

        for(iCnt = 2; iCnt < iNo; iCnt++)
        {
            if((iNo % iCnt) == 0)
            {
                bFlag = false;
                break;
            }
        }
        return bFlag;
    }
}

class program44
{
    public static void main(String Arg[])
    {
        Scanner obj = new Scanner(System.in);
        int iValue = 0;
        boolean bRet = false;

        System.out.println("Enter the number :");
        iValue = obj.nextInt(); 

        Arithematic aobj = new Arithematic(iValue);
        bRet = aobj.CheckPrime();

        if(bRet == true)
        {
            System.out.println("Number is prime ");
        }
        else
        {
            System.out.println("Number is not prime ");
        }
    }

}